test/index.ts

```diff
@@ -1,15 +1,21 @@
 import * as request from 'supertest';
 import { startServer } from '../src/setup';
 import { expect } from 'chai';
+import * as dotenv from 'dotenv';
+import { queryCreateUser } from './queries';
+import { User } from '../src/entity/user';
+import * as bcrypt from 'bcrypt';
 
 before(async () => {
+  dotenv.config({ path: __dirname + '/../test.env' });
   await startServer();
+  const userRepository = User.getRepository();
+  userRepository.clear();
 });
 
-describe('Print test', () => {
-  it('Printing', () => {
-    console.log('Test ran');
-  });
+after(() => {
+  const userRepository = User.getRepository();
+  userRepository.clear();
```

Comment:
acredito que essa função é assíncrona e ai iria precisar de um `await` pra garantir.

Reply:
Essa não é, segue assinatura:
`(method) BaseEntity.getRepository<User>(this: ObjectType<User>): Repository<User>`

Reply:
eu to falando da `clear()` :thinking:

Reply:
Ah desculpa, essa é mesmo, valeu! adicionei o await